File: Dart/que_1.dart
  // Program to interchange first and last elements in a list

import 'dart:io';

List interchange(List list) {
  int size = list.length;

  int temp = list[0];
  list[0] = list[size - 1];
  list[size - 1] = temp;

  return list;
}

void main() {
  print("Enter the size of list: ");
  int size = int.parse(stdin.readLineSync()!);

  List list = List.generate(
    size,
    (genreator) {
      print("Enter the list element");
      int data = int.parse(stdin.readLineSync()!);
      return data;
    },
  );
  print("Before Interchange this list is: $list");
  List newList = interchange(list);
  print("After Interchagne the list is: $newList");
}

File: Python/que_1.py
# Program to Interchange first and last element in list 
def interchange(newlist):
    size = len(newlist)
    
    temp = newlist[0]
    newlist[0] = newlist[size - 1]
    newlist[size - 1] = temp
    return newlist

size = int(input("Enter the size of the list: "))

element = [int(input(f"Enter the {i + 1} element: ")) for i in range(size)]

print(f"Before Interchange the list is {element}")

listData = interchange(element)

print(f"After Interchage the list is {listData}")